jjc

```diff
@@ -69,14 +69,12 @@ BEGIN {
 	support["prefix"]      = "@+"
 	support["statusmsg"]   = ""
 
-	# Shell escape the logdir path.  Because all variables used in
-	# BEGIN will have global scope, an array variable is used.  That
-	# way it can easily be emptied later.
-	local["path"]=shell_escape(irc["logdir"] "/channels")
+	# Because all variables used in BEGIN will have global scope, an array
+	# variable is used.  That way it can easily be emptied later.
 
 	# Try to create the "channels" directory while also creating all
 	# parent directories as needed.
-	local["retval"]=system("mkdir -p -- " local["path"] " >&2")
+	local["retval"]=system("mkdir -p -- \"$IRC_DIR/$IRC_HOST/channels\" >&2")
```

Comment:
Doesn't fit within 74 columns. We could use:

    local["retval"]=
    	system("mkdir -p -- \"$IRC_DIR/$IRC_HOST/channels\" >&2")

Reply:
I think there have not really been a coding style for awk applied ever, so it is up to anyone to start one I guess...